### PR TITLE
Update Message Input to count graphemes instead of unicode characters

### PR DESCRIPTION
**Pull request by mavirex**
I installed this module, grapheme splitter, so that our message input boxes better account for emojis and foreign languages:
https://github.com/orling/grapheme-splitter

Then I updated the Message Input component to count graphemes instead of unicode characters

### CODE REVIEW

File: components/InteractiveElements/MessageInput.tsx [hunk 1]
```diff
@@ -18,8 +19,24 @@ export default function MessageInput({
   const margin =
     useSelector((state: RootState) => state.screenHeight.height) * 0.01;
   const [messageFocus, setMessageFocus] = useState(false);
+  const [messageGraphemes, setMessageGraphemes] = useState(0);
+  const [splitter] = useState(new GraphemeSplitter());
   const messageLimit = 70;
```
Comment (adam-adair) [inline components/InteractiveElements/MessageInput.tsx:24]: I like Fu's idea of moving to constants

**Comment by adam-adair** [comment]
minor comment, but I like it

**Comment by fugoto** [comment]
Just tested on iPhone and it works for me! emojis only take up 1 character spot, even with different skin tones.

File: components/InteractiveElements/MessageInput.tsx (hunk 1 above)
Comment (mavirex) [reply components/InteractiveElements/MessageInput.tsx:24]: Will do!

Comment (mavirex) [reply components/InteractiveElements/MessageInput.tsx:24]: I'm going to address this in my other PR, and merge this one now

**Pull request merged**